zipline/assets/futures.py

```diff
@@ -0,0 +1,128 @@
+from pandas import Timedelta
+
+from zipline.finance.trading import TradingEnvironment
+
+
+class FutureChain(object):
+    """ Allows users to look up future contracts.
+
+    Parameters
+    ----------
+    trading_algorithm : TradingAlgorithm
+    root_symbol : str, optional
+    as_of_date : pandas.Timestamp
+    """
+
+    def __init__(self, trading_algorithm, root_symbol='', as_of_date=None):
```

Comment:
@jfkirk I think we had discussed at some point that root_symbol should be a keyword arg. Maybe it should just be positional, since there's really no sensible default case?